docs/kind-webserver.md

```diff
@@ -0,0 +1,115 @@
+## Webserver definition
+
+The webserver builder can be used to create a Docker image of a webserver. It
+supports two different ways to invoke it:
+
+1. alone, with its own zbuildfile (e.g. if you want to only serve an API with
+no assets) ;
+2. integrated within another builder (e.g. to add assets built by another
+zbuildfile) ;
+
+## Syntax
+
+zbuildfiles with webserver types have following structure:
+
+```yaml
+kind: webserver
+
+webserver: <string>
+system_packages: <map[string]string>
+config_file: <string>
+healthcheck: <bool>
+assets: <assets>
+```
+
+##### `webserver` - default: `nginx`
+
+This parameter defines which webserver you want to use for this image. Only
+`nginx` is supported for now.
+
+##### `system_packages` - not required
+
+This parameter can be used to install custom system packages in the image. It's
+a map of package names as keys and version constraints as values.
+
+Example:
+
+```yaml
+# syntax=akerouanton/zbuilder:<tag>
+kind: webserver
+
+system_packages:
+  curl: *
+```
+
+System packages are pinned to a specific version in the lockfile with the help
+of `zbuild update`. See [here](https://github.com/NiR-/zbuild#2-create-or-update-the-lock-file) for more details.
```

Comment:
`https://github.com/NiR-/zbuild#2-create-or-update-the-lock-file` => `/README.md#2-create-or-update-the-lock-file` ?

will use the repo structure instead of https links